### ISSUE TITLE
Two more copy-pasted `params.v = "3.0"` lines in `vk_ph_comms`.

### ISSUE THREAD

**Issue by deltaidea** [vk-x/vk-x-legacy#117]
One more copy-pasted `params.v = "3.0";` in `vk_ph_comms`.

See `vk_ph_comms.get_data()`, `users_load()` helper:

``` JavaScript
            if (uids.length==0){
               photos_load();
               return;
            }
            params.v = "3.0";
            app.vkApi.request({
            ...
```

It fails with message `Uncaught ReferenceError: params is not defined`.

**Comment by deltaidea** [comment]
Also one more a little above:

``` JavaScript
            params.v = "3.0";
             app.vkApi.request({
               method: "photos.getById",
               data: {photos:pids.join(','), v: "3.0" },
               ...
```

**Comment by deltaidea** [comment]
This bug was introduced in 8b3587406b82a69699e7d42591fa938664671804, after [`vkApi`](https://github.com/deltaidea/happy/blob/master/source/vk-api.litcoffee) module was first implemented in 71ab0efed61f2d61fc7e8aa9394a60cc2f3cae64...8def03ea3b720776079f4a44c9f7fc94caee4c43.

Similar bug fix: 1f47699a876f43df4395179c76cea2b32570c441.
